### DISCUSSION TITLE
File .env is not created created

### DISCUSSION THREAD

**Discussion by DimaKozhanovsky** [open-tutorials/cypress#63] [Q&A] [answered]
Colleagues, I'm doing a tutorial "Deploy project Conduit locally" sixth paragraph https://www.epic1h.com/build_conduit.

My expectations, this is a creation of the file `~/backend/.env` after inputing command `cp .env.example .env`.

**Nevertheless file is not created.**

*Please , help me to find out  the reason of issue ? Thank you in advance 🙏*

<img width="581" alt="Screen Shot 2023-02-18 at 17 24 46" src="https://user-images.githubusercontent.com/74324758/219871046-acf19196-9c42-4c96-8389-bbede1800309.png">

**Comment by DimaKozhanovsky** [comment, marked as answer]
Hey Dima! The .env file is hidden, to see this file on Mac in Terminal use `cat .env`

**Reply by breslavsky** [reply]
@DimaKozhanovsky please write `ls -a` to see all hidden files.
